ngi_pipeline/engines/piper_ngi/launchers.py

```diff
@@ -131,21 +131,22 @@ def analyze(project, sample,
                     if not keep_existing_data:
                         remove_previous_sample_analyses(project)
                 elif level == "genotype":
-                    remove_previous_genotype_analyses(project)
+                    if not keep_existing_data:
+                        remove_previous_sample_analyses(project)
```

Comment:
is this call correct or should the call in fact be to `remove_previous_genotype_analyses`?